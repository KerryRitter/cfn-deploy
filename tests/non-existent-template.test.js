const lib = require('../');


describe('CloudFormation.validateTemplate', () => {
  it('should error on non-existent template', (done) => {
    lib({
      region: 'us-east-1',
      stackName: 'test-stack',
      template: './tests/templates/non-existent-template.yaml',
    }).on('ERROR', (err) => {
      expect(err.message).toMatchSnapshot();
      done();
    });
  });
});
