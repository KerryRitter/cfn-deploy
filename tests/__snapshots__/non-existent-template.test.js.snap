// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CloudFormation.validateTemplate should error on non-existent template 1`] = `"ENOENT: no such file or directory, open './tests/templates/non-existent-template.yaml'"`;
